node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoreCollector.java

```diff
@@ -29,7 +30,9 @@
     private static final Pattern CORE_GENERATOR_PATH_PATTERN = Pattern.compile("^Core was generated by `(?<path>.*?)'.$");
     private static final Pattern EXECFN_PATH_PATTERN = Pattern.compile("^.* execfn: '(?<path>.*?)'");
     private static final Pattern FROM_PATH_PATTERN = Pattern.compile("^.* from '(?<path>.*?)'");
-    static final String GDB_PATH = "/opt/rh/devtoolset-10/root/bin/gdb";
+    static final String GDB_PATH = Files.exists(Path.of("/opt/rh/gcc-toolset-10/root/bin/gdb")) ?
```

Comment:
This checks if the file is present on the host, but it should probably check if the file is present in the container.

Reply:
yes, you might be right about that. I guess @freva can tell.

Reply:
Yes, this need to exist in the container.

Reply:
Fixed by probing the container.

Reply:
Note that old containers having devtoolset-9 or gcc-toolset-9 versions of gdb will be considered as having gcc-toolset-10 version of gdb.

Reply:
So this would fail without this change as well when we have a mix of devtoolset-9 and devtoolset-10. I will implement a check that makes this work in both cases.